libs/dh/message-archive/feature-log-search/src/lib/drawer/dh-message-archive-drawer.component.ts

```diff
@@ -104,7 +104,7 @@ export class DhMessageArchiveDrawerComponent implements OnInit, OnDestroy {
     this.drawer.open();
 
     if (this.message) {
-      this.getDocument(this.message?.messageId);
+      this.getDocument(this.message?.id);
```

Comment:
nitpick: `?` not necessary since `id` will always contain a value.

```suggestion
      this.getDocument(this.message.id);
```